test/Microsoft.ComponentDetection.Detectors.Tests/SPDX22ComponentDetectorTests.cs

```diff
@@ -71,6 +71,13 @@ public async Task TestSbomDetector_SimpleSbomAsync()
             ""licenseDeclared"": ""NOASSERTION"",
             ""copyrightText"": ""NOASSERTION"",
             ""versionInfo"": ""1.0.0"",
+            ""externalRefs"": [
+                {
+                    ""referenceCategory"": ""PACKAGE-MANAGER"",
+                    ""referenceType"": ""purl"",
+                    ""referenceLocator"": ""pkg:nuget/openssl@2.10.0""
```

Comment:
it seems that our verification tests had some extra junk after the purl causing our verification tests to break :/